### ISSUE TITLE
p4a --requirements ignores absolute folder paths

### ISSUE THREAD

**Issue by JonasT** [kivy/python-for-android#1487]
p4a's `--requirements` option appears to ignore/drop absolute folder paths. This is the output at the end building the dependencies I get:

```
[INFO]:    # Installing pure Python modules
[INFO]:    The requirements (Cython, https://github.com/JonasT/nettools/archive/master.zip, https://github.com/JonasT/wobblui/archive/master.zip, pyphen, python-dateutil, python-docx) don't have recipes, attempting to install them with pip
```
... the absolute folder path I put is simply missing. It is also not part of the site-packages as it is supposed to be in the final `.apk` created.

### Versions

* Python: python 3 host, python3crystax target
* OS: ubuntu docker
* Kivy: not used
* Cython: latest from pip

### buildozer.spec

not used

### Logs

```
[INFO]:    NDK is crystax, skipping biglink (will this work?)
[INFO]:    # Postbuilding recipes
[INFO]:    Postbuilding harfbuzz for armeabi-v7a
[INFO]:    Postbuilding hostpython3crystax for armeabi-v7a
[INFO]:    Postbuilding jpeg for armeabi-v7a
[INFO]:    Postbuilding libxml2 for armeabi-v7a
[INFO]:    Postbuilding openssl for armeabi-v7a
[INFO]:    Postbuilding png for armeabi-v7a
[INFO]:    Postbuilding sdl2_image for armeabi-v7a
[INFO]:    Postbuilding sdl2_mixer for armeabi-v7a
[INFO]:    Postbuilding sdl2_ttf for armeabi-v7a
[INFO]:    Postbuilding freetype for armeabi-v7a
[INFO]:    Postbuilding libxslt for armeabi-v7a
[INFO]:    Postbuilding python3crystax for armeabi-v7a
[INFO]:    Postbuilding lxml for armeabi-v7a
[INFO]:    Postbuilding reportlab for armeabi-v7a
[INFO]:    Postbuilding sdl2 for armeabi-v7a
[INFO]:    Postbuilding setuptools for armeabi-v7a
[INFO]:    Postbuilding six for armeabi-v7a
[INFO]:    Postbuilding Pillow for armeabi-v7a
[INFO]:    Postbuilding pyjnius for armeabi-v7a
[INFO]:    Copying pyjnius java class to classes build dir
[INFO]:    -> directory context /root/.local/share/python-for-android/build/other_builds/pyjnius-python3crystax-sdl2/armeabi-v7a__ndk_target_19/pyjnius
[DEBUG]:   -> running cp -a jnius/src/org /root/.local/share/python-for-android/build/javaclasses/unnamed_dist_1
[INFO]:    <- directory context /root/.p4a-shell-build
[INFO]:    Postbuilding pysdl2 for armeabi-v7a
[INFO]:    # Installing pure Python modules
[INFO]:    The requirements (Cython, https://github.com/JonasT/nettools/archive/master.zip, https://github.com/JonasT/wobblui/archive/master.zip, pyphen, python-dateutil, python-docx) don't have recipes, attempting to install them with pip
[INFO]:    If this fails, it may mean that the module has compiled components and needs a recipe.
...
```
As mentioned above, there should be an absolute folder path in that pure python modules list, and there just isn't. It is just quietly dropped which breaks the final `.apk` since the dependency is just missing.

**Comment by JonasT** [comment]
Much earlier, I get this output:

```[INFO]:    Dist will also contain modules (python-docx, /my-app-lib-copy/, https://github.com/JonasT/wobblui/archive/master.zip, python-dateutil, https://github.com/JonasT/nettools/archive/master.zip, pyphen, Cython) installed from pip
```
So the absolute path dependency is initially recognized, and gets lost somewhere.

**Comment by JonasT** [comment]
This is now fixed on p4a master with the merge of #1489 ! :tada:
